spec/Manager/ProviderManagerSpec.php

```diff
@@ -12,11 +12,14 @@
 
 final class ProviderManagerSpec extends ObjectBehavior
```

Comment:
Why change this file at all? It's the spec for the `ProviderManager`

Reply:
isEnabled needed to be set